src/test/java/com/keboola/tableexporter/GetTablesTest.java

```diff
@@ -0,0 +1,35 @@
+package com.keboola.tableexporter;
+
+import com.keboola.tableexporter.exception.ApplicationException;
+import org.apache.commons.io.FileUtils;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Paths;
+
+import static org.junit.Assert.assertTrue;
+
+public class GetTablesTest extends BaseTest {
+
+    @Test
+    public void testGetTables() throws IOException, URISyntaxException, ApplicationException {
```

Comment:
it's missing table setup so it relies on the database state

Reply:
It assumes an empty demo DB.  The tables used for the export tests are generated and dropped during those tests

Reply:
must be blind, but i don't see it where it's generated :)

Reply:
It's not, it's a built in part of the test DB image:)

Reply:
yeah, that's what i'm saying - you can't run the tests against an aws oracle, also it's inconsistent because large table is setup in the tests - but i'd probably make this as another issue

Reply:
hmmm, I see,  I'm not sure how to resolve this.  I suppose it should drop all those sample tables on setup?  Thanks, I'll make the issue.